.github/workflows/nodejs.yml

```diff
@@ -7,16 +7,16 @@ on:
   push:
     branches:
       - master
-  
 
+      
 jobs:
   build:
 
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
-        node-version: [8.x, 10.x, 12.x]
```

Comment:
Because we use `10.16.3`, 8.x doesn't have to be supported.
12.x should be tested for our future possible upgrade.